### PR TITLE
fix: Farm card and row liquidity and details translations

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/views/Farms/components/FarmCard/CardActionsContainer.tsx [hunk 1]
```diff
@@ -68,15 +68,17 @@ const CardActions: React.FC<FarmCardActionsProps> = ({ farm, account, addLiquidi
     )
   }
 
+  const tokenEarnedString = TranslateString(330, 'CAKE Earned', { asset: 'CAKE' }).split(' ')
```
Comment (RabbitDoge) [inline src/views/Farms/components/FarmCard/CardActionsContainer.tsx:71]: You don't need to use a translation here , furthermore, splitting like doesn't guarantee to return an acceptable result. What takes 2 words in English, can take 1 or 3 in other langs.

After the tokens reworks, you can get the token symbol with `farm.token` and `farm.quoteToken`. Have a look at the `/state/types` file

**Review by RabbitDoge — CHANGES_REQUESTED** [review]

File: src/views/Farms/components/FarmCard/CardActionsContainer.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Farms/components/FarmCard/CardActionsContainer.tsx:71]: 👍🏻 sure

Comment (memoyil) [reply src/views/Farms/components/FarmCard/CardActionsContainer.tsx]: We don't have earned token info in farm object therefore I reverted that code stub

**Review by memoyil — COMMENTED** [review]
@RabbitDoge Can you review it again

File: src/components/ExpandableSectionButton/ExpandableSectionButton.tsx [hunk 2]
```diff
@@ -19,10 +20,12 @@ const Wrapper = styled.div`
 `
 
 const ExpandableSectionButton: React.FC<ExpandableSectionButtonProps> = ({ onClick, expanded }) => {
+  const TranslateString = useI18n()
+
   return (
     <Wrapper aria-label="Hide or show expandable content" role="button" onClick={() => onClick()}>
       <Text color="primary" bold>
-        {expanded ? 'Hide' : 'Details'}
+        {expanded ? 'Hide' : TranslateString(658, 'Details')}
```
Comment (hachiojidev) [inline src/components/ExpandableSectionButton/ExpandableSectionButton.tsx:28]: Please wrap `Hide` in the translate function as well. If we don't have an ID already just use 999

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx [hunk 3]
```diff
@@ -42,11 +42,13 @@ const HarvestAction: React.FunctionComponent<FarmWithStakedValue> = ({ pid, user
...
 
+  const tokenEarnedString = TranslateString(330, 'CAKE Earned', { asset: 'CAKE' }).split(' ')
+
   return (
     <ActionContainer>
       <ActionTitles>
-        <Title>CAKE </Title>
-        <Subtle>EARNED</Subtle>
+        <Title>{tokenEarnedString[0]} </Title>
+        <Subtle>{tokenEarnedString[1]}</Subtle>
```
Comment (hachiojidev) [inline src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx:51]: In this case it is okay to leave `CAKE` as is and just wrap `Earned` in the translate string
```
{TranslateString(999, 'Earned')}
```

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx [hunk 4]
```diff
@@ -42,11 +42,13 @@ const HarvestAction: React.FunctionComponent<FarmWithStakedValue> = ({ pid, user
     updateValue.current(earningsBusd)
   }, [earningsBusd, updateValue])
 
+  const tokenEarnedString = TranslateString(330, 'CAKE Earned', { asset: 'CAKE' }).split(' ')
```
Comment (RabbitDoge) [inline src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx:45]: cake doesn't need to be translated, and this split can lead to unexpected behaviour. Please remove and just translate `EARNED`

**Review by RabbitDoge — CHANGES_REQUESTED** [review]

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx (hunk 3 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx]: Done

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx (hunk 4 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx]: Done

File: src/components/ExpandableSectionButton/ExpandableSectionButton.tsx (hunk 2 above)
Comment (memoyil) [reply src/components/ExpandableSectionButton/ExpandableSectionButton.tsx]: Done

**Review by hachiojidev — APPROVED** [review]

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**